pages/index.js

```diff
@@ -8,13 +12,27 @@ export default function Home() {
 	const Form = dynamic(() => import('../src/components/Form/Form'), {
 		ssr: false, // This line is important.
 	});
+
+	const archiveTasks = useStore(state => state.archiveTasks);
+
 	return (
 		<main>
 			<h1 hidden>ToNotDo</h1>
-			<h2>Create</h2>
-			<Form editMode={false} />
-			<h2>ToDo</h2>
+			<h2 hidden>Create</h2>
+			<Form editMode={false} className="header" />
+			<h2 hidden>ToDo</h2>
```

Comment:
I think using styled-components plus classes at the same time makes reading and debugging the code quite confusing (at least if it isn't utility classes only). I REALLY recommend adding a refactoring user story very soon in which you add all your class stylings to the corresponding styled-components. We can also talk about it before you start the refactoring.